api/src/pcapi/routes/backoffice/pro/blueprint.py

```diff
@@ -279,6 +279,7 @@ def create_offerer() -> utils.BackofficeResponse:
         motorDisabilityCompliant=None,
         visualDisabilityCompliant=None,
         comment=None,
+        isOpenToPublic=None,
```

Comment:
Cet endpoint est réservé à la création de structure pour les entreprises « non-diffusibles », donc l'adresse n'est pas récupérable par l'API Sirene, et n'est pas stockée. Le blocage pour les diffusibles se fait à la vérification du formulaire : https://github.com/pass-culture/pass-culture-main/blob/v323.0.4/api/src/pcapi/routes/backoffice/pro/forms.py#L137
Par conséquent, sans adresse, il n'y a forcément pas d'accueil du public, on peut directement pré-remplir à `False` :
```suggestion
        isOpenToPublic=False,
```

Reply:
En fait dans notre chantier de régularisation, on souhaite utiliser la valeur de ce champ pour savoir l'état d'une `Venue`: 
- `True` => hérité de isPermanent
-  `False` => régularisée
- `None` => pas encore traitée

Reply:
ok pour moi